src/modules/drawing/components/useDrawingModeInteraction.composable.js

```diff
@@ -26,6 +29,7 @@ export default function useDrawingModeInteraction({
     useGeodesicDrawing = false,
     snapping = false,
     drawEndCallback = null,
+    startingFeature = null,
```

Comment:
might be good to also add some JSDoc on what types are expected here, it starts to get quite big...